e2e/scripts/st_file_uploader.py

```diff
@@ -56,3 +56,20 @@
         st.text("No upload")
     else:
         st.text(form_file.read())
+
+
+if st._is_running_with_streamlit:
```

Comment:
I assume this `if` line is here because we run e2e tests in "bare Python mode" as part of our Python tests, and this test doesn't work in that circumstance? Can you add a comment here explaining that?